docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -84,54 +90,64 @@ Register the BIG-IP Controller as a Service Broker
    You'll need to provide the following information:
 
    - a name for the Service Broker;
-   - a username and password for a user with permission to interact with the Service Broker API (sames as in the :code:`bigip.status` section of the Application Manifest);
+   - a username and password for a user that can interact with the Controller's API (this can be the same user provided in the :code:`bigip.status` section of the Application Manifest);
    - the Route created in the previous step.
 
    .. code-block:: console
 
       cf create-service-broker cfbigip-sb someUser somePass https://sbtest.example.com
 
-#. `Enable Service access`_ for the f5servicebroker Service.
+#. `Enable Service Access`_ for the :code:`f5servicebroker` Service.
 
    .. code-block:: console
 
       cf enable-service-access f5servicebroker
 
 #. `Create a Service instance`_ for your users.
 
-   .. tip:: The plan name should match that provided in your Application Manifest.
+   .. tip:: Provide a Plan name that matches a Plan defined in your Application Manifest.
 
    .. code-block:: console
 
-      cf create-service f5servicebroker sbtest <instance_name>
+      cf create-service f5servicebroker <plan_name> <instance_name>
 
 #. Verify availability of the Service in the Cloud Foundry Marketplace.
 
    .. code-block:: console
 
       cf marketplace -s f5servicebroker
 
-How to Use the f5servicebroker Service
---------------------------------------
+Bind the f5servicebroker Service to a Route
+-------------------------------------------
 
-Once an administrator has completed the section above, developers can use the :code:`f5servicebroker` Service for their Apps. This is done by binding the Service to the App's Route.
+Once an administrator has completed the section above, developers can use the :code:`f5servicebroker` Service for their Apps. To do so, bind the :code:`f5servicebroker` Service to the App's Route.
 
 .. code-block:: console
 
-   cf bind-route-service example.com sbtest --hostname myApp
+   cf bind-route-service example.com <instance_name> [--hostname HOSTNAME] [--path PATH]
+
+When you `bind a Route to the Service`_, the |cfctlr| creates a virtual server, pool(s), and pool member(s) on the BIG-IP device with the requested policy(ies) and profile(s) attached.
```

Comment:
It will only do this if the route being bound to is mapped to an app that has endpoints, you can bind to an unmapped route and nothing will show on the BIG-IP